cloudrun-malware-scanner/cloudbuild.yaml

```diff
@@ -0,0 +1,35 @@
+steps:
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'build'
+      - '-t'
+      - "${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:$COMMIT_SHA"
+      - '-t'
+      - '${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:latest'
+      - '-f'
+      - 'cloudrun-malware-scanner/Dockerfile'
+      - 'cloudrun-malware-scanner'
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'push'
+      - "${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:$COMMIT_SHA"
+  - name: '${_GCR_HOSTNAME}/cloud-builders/docker'
+    args:
+      - 'push'
+      - '${_GCR_HOSTNAME}/$PROJECT_ID/${_SERVICE_NAME}:latest'
+  - name: '${_GCR_HOSTNAME}/google.com/cloudsdktool/cloud-sdk'
+    entrypoint: 'gcloud'
+    args:
+      - 'run'
+      - 'deploy'
```

Comment:
can u change
"deploy"
to
"run services update"

deploy will automatically create a new cloud run service if it doesnt exist
i would prefer this to fail if existing service does not exist

Reply:
Done. We can discuss this a little in our sync.
My thought is, we might need both. like borg start and borg update.
The point is, we shouldn't let people to start a service with his/her own account. Instead, we have the build account already, all deploy should be done this this build account. Thus, we need the deploy as well (or we can do delegation, I'm not sure how to do it in GCP yet, but there should be a way to support that).